src/main/cpp/t08_fibb.cpp

```diff
@@ -21,5 +21,21 @@
 using namespace std;
 
 int t08_fibb() {
-
+	int N, i = 0, a = 1, b = 1;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'b' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513864)